### DISCUSSION TITLE
Extend theme colors key contains number

### DISCUSSION THREAD

**Discussion by chocotree** [unocss/unocss#1972] [Q&A] [answered]
Hi, does config key contains number currently not allowed ?

[Playground example](https://uno.antfu.me/play/?html=DwCwjABAxgNghgZwQXgEQBcCmAPdBaABwEsA7Aa0ICciBbOSgT1QD4AoCCACUxhgHsIAVRJ8AwgGVxrYAHpwbaQWjwkaLLkKkKBanUYAmFuwjFyEEeggB3PpTIRAfBuAF3ekyCC4EtiIUGHPlMKEQA5AFcaACNMSiMOQOtbMlcABWYgA&config=JYWwDg9gTgLgBAbwFBzgEwKYDNgDsMDCEuOA5gDQpxhQYDOGMAqrhJQL5xZQQhwDkAV1YBjOnX5IkGAB6RY6bAENBAG3iYc%2BIiWCkAFMlQ16jOgC44AbSrHaDZq30BKSqgC6buDAAWGEBiWRqhwIhCq0BaItnAAkGB4ANZBMSFxNKBKUACelvwAxBgAHAAsAJxFZfxeaWnxUJk5AEx5%2BWgAzACsRUUA7NWptbGsAHKCIABGGFCtHd19A7Uh7DHsXivszkhAA&options=N4XyA)

I want to refactor my small project from WindiCSS into UnoCSS and found out theme colors key contain number not work.

**Comment by chocotree** [comment]
Here is the playground example config.
![config](https://user-images.githubusercontent.com/33096368/205863414-e42ecc60-c3a7-4f1b-93ea-3e60b30cc629.jpeg)

**Comment by zyyv** [comment, marked as answer]
You should change to this, [Playground](https://uno.antfu.me/play/?html=DwEwlgbgBALgpgDxgWgA5gHYGs0CcwC2AhrgJ6yIoDGcG8uAfAFBRQD2GyIcxGIUAdqiYAzjCJ1BUAGYBXADbzyVWWLYEwALyIAjeXCbAA9OAjNQkCkjSYcqfMTIAmK9Vr1mrDlx4T%2BQ0XFJITlFZVUYdS1dfUMTSGYgA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmANHGFKgM6owCCMMUwARgK4zDoCeZF1tAJIBjAlT6UaMAKpYIcAL5x0UCCDgByNnOFUqGgFAHUAD0iwUGAIZsANvDSYc%2BQsCIAKBAbhwYAC1QQVAAuRG8fOFFbaCpQrwiIsGwAazjwhJ8KUCsobjSMjIARAFEAMTppABkAFVCNAGJUAA4AFgBOJraNEnSCuAAmOvrkAGYAViamgHZu3oSFHoKF3uWfVfJJWli4AG1%2BKVkIdwBKCQF6RmZ2Th4TgF0ehWODIA&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwF8g)
```ts
colors: {
  pink: {
    primary: {
      DEFAULT: '#e84989',
      2: '#d35887'
    }
  }
}
```
The reason is that when we parse the color key, letters and numbers will be separated.
